src/Service.GraphQLBuilder/GraphQLUtils.cs

```diff
@@ -209,24 +212,51 @@ public static ObjectType UnderlyingGraphQLEntityType(IType type)
         }
 
         /// <summary>
-        /// Parse a given string value to supported GraphQL Type and GraphQLValueNode
+        /// Translates a JSON string or number value defined in the runtime configuration to a GraphQL {Type}ValueNode which represents
+        /// the associated GraphQL type. The target value type is referenced from the passed in parameterDefinition which
+        /// holds database schema metadata.
         /// </summary>
-        public static Tuple<string, IValueNode> GetGraphQLTypeAndNodeTypeFromStringValue(string stringValue)
+        /// <param name="defaultValueFromConfig">String representation of default value defined in runtime config.</param>
+        /// <param name="parameterDefinition">Database schema metadata for stored procedure parameter which include value and value type.</param>
+        /// <returns>Tuple where first item is the string representation of a GraphQLType (e.g. "Byte", "Int", "Decimal")
+        /// and the second item is the GraphQL *type*ValueNode </returns>
```

Comment:
> GraphQL *type*ValueNode

did you mean {Type}ValueNode? I guess I am not entirely sure what *type*ValueNode means if not. Just that this is of Type ValueNode?

Reply:
Yes you're correct about my meaning  {Type}ValueNode

Reply:
updated